apps/ejabberd/src/mod_mam_cassandra_arch.erl

```diff
@@ -16,7 +17,7 @@
 %% MAM hook handlers
 -export([archive_size/4,
          archive_message/9,
-         lookup_messages/15,
+         lookup_messages/3,
```

Comment:
I know there is `archive_message/9` it wasn't my goal to change this in this PR.